src/messages/javascript.js

```diff
@@ -239,3 +239,30 @@ export const KNOWN_LIBRARY = {
     'testcases_content', 'test_packed_packages', 'blacklisted_js_library',
   ],
 };
+
+function deprecatedAPI(api) {
+  return {
+    code: apiToMessage(api),
+    legacyCode: [
+      'js',
+      'deprecated',
+      api,
+    ],
+    message: _(`"${api}" is deprecated or unimplemented`),
+    description: _(singleLineString`APIs that have been deprecated by Chrome
```

Comment:
I would change this to `This API has been deprecated by Chrome and has not been implemented by Firefox`. That is more in line with other descriptions.